great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -195,7 +199,7 @@ def _build_parameters(
             parameters=parameters,
         )
         metric_values: Union[
-            np.ndarray, List[Union[Any, Number]]
+            np.ndarray, List[Union[Any, Number, List[Number]]]
```

Comment:
✅

Reply:
@jcampbell Yes, adding the `List[Number]` is appropriate (but only one small step towards accommodating metrics which return results of varying dimensions).

Reply:
I figure it's all redundant if we have Any there...